packages/cli/src/utils/directory.test.ts

```diff
@@ -8,77 +8,210 @@ const pathsToMatch = (expected: string, desired: string) => {
   return expectedResolved === desiredResolved
 }
 
-describe('directory with base path', () => {
+describe('withBasePath as the current dir `.`', () => {
+  const basePath = '.'
+
   describe('coreDir', () => {
-    it('is the fastsstoreDir + core', () => {
-      expect(pathsToMatch(coreDir, './node_modules/@faststore/core')).toBe(true)
+    it('is the faststoreDir + core', () => {
+      const { coreDir: basedCoreDir } = withBasePath(basePath)
+
+      expect(pathsToMatch(basedCoreDir, './node_modules/@faststore/core')).toBe(true)
+    })
+
+    describe('when is in a monorepo', () => {
+      it.todo('can look at its parent until it reaches the monorepo directory')
     })
   })
 
   describe('tmpDir', () => {
-    it('is the cwd + .faststore', () => {
-      expect(pathsToMatch(tmpDir, './.faststore')).toBe(true)
+    it('is the basePath + .faststore', () => {
+      const { tmpDir: basedTmpDir } = withBasePath(basePath)
+
+      expect(pathsToMatch(basedTmpDir, './.faststore')).toBe(true)
     })
   })
 
   describe('userDir', () => {
     it("returns the directory of the starter's package.json", () => {
-      expect(pathsToMatch(userSrcDir, './src')).toBe(true)
+      const { userSrcDir: userSrcDirWithBase } = withBasePath(basePath)
+
+      expect(pathsToMatch(userSrcDirWithBase, './src')).toBe(true)
     })
   })
 
   describe('tmpCustomizationsSrcDir', () => {
     it('returns the directory of customizations directory on the tmp dir', () => {
-      expect(pathsToMatch(tmpCustomizationsSrcDir, './.faststore/src/customizations/src')).toBe(true)
+      const { tmpCustomizationsSrcDir: tmpCustomizationsSrcDirWithBase } = withBasePath(basePath)
+
+      expect(pathsToMatch(tmpCustomizationsSrcDirWithBase, './.faststore/src/customizations/src')).toBe(true)
     })
   })
 
   describe('userThemesFileDir', () => {
     it("returns the directory of the starter's theme directory", () => {
-      expect(pathsToMatch(userThemesFileDir, './src/themes')).toBe(true)
+      const { userThemesFileDir: userThemesFileDirWithBase } = withBasePath(basePath)
+
+      expect(pathsToMatch(userThemesFileDirWithBase, './src/themes')).toBe(true)
     })
   })
 
   describe('tmpThemesCustomizationsFile', () => {
     it('returns the path of the theme file on the .faststore dir', () => {
-      expect(pathsToMatch(tmpThemesCustomizationsFile, './.faststore/src/customizations/src/themes/index.scss')).toBe(true)
+      const { tmpThemesCustomizationsFile: tmpThemesCustomizationsFileWithBase } = withBasePath(basePath)
+
+      expect(pathsToMatch(tmpThemesCustomizationsFileWithBase, './.faststore/src/customizations/src/themes/index.scss')).toBe(true)
     })
   })
 
   describe('tmpCMSDir', () => {
-    it('returns the path of the CMS dir on the .faststore dir', () => {
-      expect(pathsToMatch(tmpCMSDir, './.faststore/cms/faststore')).toBe(true)
+    it('returns the path of the theme file on the .faststore dir', () => {
```

Comment:
question: I believe it was changed by mistake
```suggestion
    it('returns the path of the CMS dir on the .faststore dir', () => {
```